client/src/pages/profile/Profile.jsx

```diff
@@ -57,6 +62,15 @@ export default function ProfilePage({userData, isLoggedIn}) {
         }
     }
 
+    const handleFilter = (value) => {
+        axios.post(`http://localhost:3000/reviews/landlordReviews/${landlordId.landlord_id}`,{
```

Comment:
drop localhost here and rely on the proxy settings from webpack dev server config. Not sure how this would affect the production build though.